test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/RemoteAttributeTest.cs

```diff
@@ -1041,8 +1039,7 @@ private static ServiceCollection GetServiceCollection(IStringLocalizerFactory lo
             var serviceCollection = new ServiceCollection();
             serviceCollection
                 .AddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>()
-                .AddSingleton<ILoggerFactory>(new NullLoggerFactory())
-                .AddSingleton<UrlEncoder>(new UrlTestEncoder());
+                .AddSingleton<ILoggerFactory>(new NullLoggerFactory());
```

Comment:
No need to change this.

Reply:
No need to keep it hanging around if its original intended use was for routing. If we need it we can add in future.